pwa/src/templates/organizationDetail/OrganizationDetailTemplate.module.css

```diff
@@ -36,9 +36,28 @@
   margin-block-end: var(--web-app-size-xs);
 }
 
+.headerContent {
+  flex: 2;
+}
+
+.logoContainer {
+  display: flex;
+  align-items: center;
```

Comment:
I'm unsure if you need these rules.

(Also: I would imagine that this is better outlined `flex-start`, which would be the same as not stating any rule at all).